src/Microsoft.DocAsCode.HtmlToPdf/ConvertWrapper.cs

```diff
@@ -36,9 +36,13 @@ public ConvertWrapper(PdfOptions pdfOptions)
 
         #region Public Methods
 
-        public static void PrerequisiteCheck()
+        public static void PrerequisiteCheck(string filePath)
         {
-            if (!CommandUtility.ExistCommand(Constants.PdfCommandName))
+            if ((!string.IsNullOrEmpty(filePath)) && (!File.Exists(filePath)))
+            {
+                throw new FileNotFoundException($"The file specified could not be found: '{filePath}'.");
```

Comment:
Use `DocfxException`. DocFX's conversion is to wrap all the exceptions into `DocfxException` or `DocumentException`.